### PR TITLE
add hashtags to onboarding categories

### PR DESCRIPTION
**Pull request by sdubinsky**

### CODE REVIEW

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 1]
```diff
@@ -36,6 +36,9 @@
     - account: '@jquillin@fashionsocial.host'
       type: 'account'
       summary: Glass Magazine Fmr Fashion editor.
+    - hashtag: "#catsofmastodon"
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:39]: Single quotes, or double? Should pick one and standardize.

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 2]
```diff
@@ -36,6 +36,9 @@
...
+      bio: cats!
+      summary: "everyone loves cats!"
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:41]: Capitalize "Everyone"

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 3]
```diff
@@ -91,6 +94,9 @@
     - account: '@craignewmark@mastodon.social'
       type: 'account'
       summary: Founder of Craigslist.
+    - hashtag: '#rubyonrails'
+      bio: 'the software Mastodon itself runs on'
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:98]: Capitalize "The", and add a period.

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 4]
```diff
@@ -91,6 +94,9 @@
...
+      summary: 'An all-in-one server framework that heavily relies on convention over configuratoin.'
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:99]: configuratoin -> configuration (typo).

File: app/lib/onboarding/v2/onboarding_categories_production.yml [hunk 5]
```diff
@@ -177,3 +183,6 @@
     - account: '@ginaperry@socel.net'
       type: 'account'
       summary: Children’s book author and illustrator.
+    - hashtag: '#mosstodon'
+      bio: lovely pictures of moss.
```
Comment (rileyhCode) [inline app/lib/onboarding/v2/onboarding_categories_production.yml:187]: Capitalize "Lovely"

**Review by rileyhCode — CHANGES_REQUESTED** [review]

File: app/lib/onboarding/v2/onboarding_categories_production.yml (hunk 2 above)
Comment (sdubinsky) [reply app/lib/onboarding/v2/onboarding_categories_production.yml]: removed this one entirely, I didn't like my captions.

**Review by rileyhCode — APPROVED** [review]
👍

**Pull request merged**